shell/browser/api/electron_api_desktop_capturer.cc

```diff
@@ -352,7 +352,7 @@ void DesktopCapturer::StartHandling(bool capture_window,
       auto capturer = MakeScreenCapturer();
       if (capturer) {
         screen_capturer_ = std::make_unique<NativeDesktopMediaList>(
-            DesktopMediaList::Type::kScreen, std::move(capturer));
+            DesktopMediaList::Type::kScreen, std::move(capturer), true, true);
```

Comment:
Failing tests are due to mixing `add_current_process_windows` with screen capturer which is not supported.